env0/resource_environment.go

```diff
@@ -381,6 +405,24 @@ func shouldUpdateTTL(d *schema.ResourceData) bool {
 	return d.HasChange("ttl")
 }
 
+func updateTemplate(d *schema.ResourceData, apiClient client.ApiClientInterface) diag.Diagnostics {
```

Comment:
can't we use the existing update template function from `resource_template`?

Reply:
It's a little different...
We need to get the environment and then get the bluePrintId lines 414-417
There's also the prefix.

I think it's pretty minimal and any future changes in template will take here as well (because we use template functions).